src/Symfony/Component/Notifier/Tests/ChatterTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Symfony\Component\Notifier\Tests;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Notifier\Chatter;
+use Symfony\Component\Notifier\Tests\Transport\DummyMessage;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+class ChatterTest extends TestCase
+{
+    /** @var MockObject&TransportInterface */
+    private $transport;
+    /** @var MockObject&MessageBusInterface */
+    private $bus;
+
+    protected function setUp(): void
+    {
+        $this->transport = $this->createMock(TransportInterface::class);
+        $this->bus = $this->createMock(MessageBusInterface::class);
+    }
+
+    public function testSendWithoutBus(): void
+    {
+        $chatter = new Chatter($this->transport);
+
+        $message = new DummyMessage();
+
+        $this->transport
+            ->expects($this->once())
+            ->method('send')
+            ->with($message);
+
+        $sentMessage = $chatter->send($message);
+
+        $this->assertNotNull($sentMessage);
+        $this->assertSame($message, $sentMessage->getOriginalMessage());
+    }
+
+    public function testSendWithBus(): void
+    {
+        $chatter = new Chatter($this->transport, $this->bus);
+
+        $message = new DummyMessage();
+
+        $this->transport
+            ->expects($this->once())
+            ->method('send')
+            ->with($message);
+
+        $this->bus
+            ->expects($this->once())
+            ->method('dispatch')
+            ->with($message);
+
+        $sentMessage = $chatter->send($message);
+
+        $this->assertNull($sentMessage);
```

Comment:
You could inline here: `$this->assertNull($chatter->send($message));` to make it shorter